docs/how_to_guides/validation/how_to_trigger_slack_notifications_as_a_validation_action.rst

```diff
@@ -1,16 +1,58 @@
 .. _how_to_guides__validation__how_to_trigger_slack_notifications_as_a_validation_action:
 
-How to trigger slack notifications as a Validation Action
+How to trigger Slack notifications as a Validation Action
 =========================================================
 
-.. admonition:: Admonition from Mr. Dickens
+This guide will help you trigger Slack notifications as a Validation Action.  If you have validation suites that take a very long time to run, it will allow you to receive a Slack message as to whether or not the Validation succeeded or failed.
```

Comment:
```suggestion
This guide will help you trigger Slack notifications as a :ref:`Validation Action <validation_actions>`_
.  It will allow you to send a Slack message including information about a Validation Result, including whether or not the Validation succeeded.
```

NOTE: you'll also need to add `.. _validation_action:` to just above the "ValidationActions" section in the validation_operators_and_actions.rst file in reference.